.github/ISSUE_TEMPLATE/issue_template.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Issue Template
+about: File an issue
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!--
+This is a suggested issue template for tedana.
+
+If there is other information that would be helpful to include, please don not hesitate to add it!
+
+Before submitting, please check to make sure that the issue is not already addressed.
+If this is a usage question, please checkout NeuroStars here:
+https://neurostars.org/
+and tag your topic with "multi-echo" so that we can see it.
```

Comment:
I don't know if we should include "so that we can see it." since I'm not sure if all folks are following that tag ! And I troll NeuroStars too often, anyway :) But I do think we should encourage tagging !

Reply:
Makes sense, removed in 533c1569e20ab9e8ae7cbfc73a9760a18b665fe6